dashboard/src/components/skills/reuseSkills/ReuseOrMoveSkillsModal.vue

```diff
@@ -14,16 +14,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 <template>
-  <b-modal :id="firstSkillId" size="lg" :title="`Reuse Skills in this Project`" v-model="show"
+  <b-modal :id="firstSkillId" size="lg" :title="`${actionName} Skills in this Project`"
+           v-model="show"
            :no-close-on-backdrop="true" :centered="true"
            header-bg-variant="info" header-text-variant="light" no-fade role="dialog" @hide="cancel"
            aria-label="'Reuse Skills in this project'">
     <skills-spinner :is-loading="isLoading"/>
 
     <div v-if="!isLoading" data-cy="reuseModalContent">
-      <no-content2 v-if="importFinalizePending" title="Cannot reuse"
-                   message="Cannot initiate skill reuse while skill finalization is pending."/>
-      <div v-if="!importFinalizePending">
+      <no-content2 v-if="importFinalizePending" :title="`Cannot ${actionName}`"
+                   :message="`Cannot initiate skill ${actionNameLowerCase} while skill finalization is pending.`"/>
+      <no-content2 v-if="state.skillsWereMovedOrReusedAlready" title="Please Refresh"
+                   message="Skills were moved or reused in another browser tab OR modified by another project administrator. Please refresh the page."/>
```

Comment:
It would be nice to have a link to refresh the skills or to do it automatically with a countdown as opposed to directing the user to refresh the page

Reply:
:+1: